jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/feedback/Snackbar.java

```diff
@@ -43,55 +59,69 @@ public void close() {
         }
     }
 
+    /**
+     * Checks if the snackbar message is of the specified type (e.g. error message, success message).
+     *
+     * @param messageType the message type to check for matching.
+     *
+     * @return {@code true} if the message type of this snackbar matches the specified message type.
```

Comment:
{@code true} if the message type of this snackbar matches given message type, otherwise {@code false}. Мы приняли, что не тсавим точки нигде, кроме как в summary джавадоков.